Mathlib/Data/Complex/Basic.lean

```diff
@@ -58,6 +58,9 @@ theorem eta : ∀ z : ℂ, Complex.mk z.re z.im = z
 theorem ext : ∀ {z w : ℂ}, z.re = w.re → z.im = w.im → z = w
   | ⟨_, _⟩, ⟨_, _⟩, rfl, rfl => rfl
 
+lemma «forall» {p : ℂ → Prop} : (∀ x, p x) ↔ ∀ a b, p ⟨a, b⟩ := by aesop
+lemma «exists» {p : ℂ → Prop} : (∃ x, p x) ↔ ∃ a b, p ⟨a, b⟩ := by aesop
+
 attribute [local ext] Complex.ext
```

Comment:
tiny gripe: please add your new lemmas after the `attribute [local ext] Complex.ext` line, to keep the attribute tag together with the lemma it refers to.

Reply:
@javierlcontreras please make this change, and merge the PR afterwards:

bors d+

Thanks for the lemmas!